tests/util/localregistry/localregistry.yaml

```diff
@@ -62,27 +58,32 @@ spec:
     port: 5000
     protocol: TCP
 ---
-apiVersion: v1
-kind: Pod
+apiVersion: extensions/v1beta1
+kind: DaemonSet
```

Comment:
Why DaemonSet ? This is pretty expensive and not needed.

Reply:
Ah, are you trying to create a local port on each node, to avoid the certificates ? 

Why not install the registry the proper way, with certificates - we have scripts and even our own CA. This will also allow to decouple the build ( I believe with localhost you must build/push from inside the cluster- with certs we can push to multiple clusters, which will be needed if we maintain multiple load test clusters)

Reply:
Sure but this is a little more involved. We are trying to validate that in cluster registry actually improves anything. As this stand this works both for minikube and gke, and is usable by anyone, using our CA would mean sharing some secrets.

Reply:
For multicluster we can keep using external docker registry.